### PR TITLE
Stab at adding some additional features to creatures

### PR DESCRIPTION
**Pull request by ELadner**
Added three additional functions to Creature plugin.

Probably not ready for a merge, but more likely for comments.  I think I got the major points, but would definitely like some experienced eyeballs to take a look.

### CODE REVIEW

**Comment by ELadner** [comment]
And.. I was a commit behind.  Should I resolve those on my side first?

**Comment by mtijanic** [comment]
Yeah, better if they're resolved on your end.

Have you tested these at all? I don't expect anything particularly weird to happen, but tests are always nice :)

File: Plugins/Creature/Creature.hpp [hunk 1]
```diff
@@ -83,6 +83,10 @@ class Creature : public NWNXLib::Plugin
     ArgumentStack LevelDown                     (ArgumentStack&& args);
     ArgumentStack SetChallengeRating            (ArgumentStack&& args);
```
Comment (mtijanic) [inline Plugins/Creature/Creature.hpp]: Oh, and while you're resolving the merge, please humor my OCD and remove this newline.

Comment (ELadner) [reply Plugins/Creature/Creature.hpp]: Lol.  Can do

**Comment by ELadner** [comment]
merged upstream changes, removed blank line.  ;)

Tests are still a WIP.

**Comment by ELadner** [comment]
ready for review.  Tests pass in a test module.  The chicken knows Stunning Fist now.

**Comment by mtijanic** [comment]
Looks great, thanks!

**Pull request merged**